RoomMe/app/src/main/java/com/example/trocket/roomme/getUsersAsync.java

```diff
@@ -10,15 +10,31 @@
 public class getUsersAsync extends AsyncTask<Integer, Void, ArrayList<User>> {
 
     HomeActivity home;
+    HomeFragment homeF;
+    int selected;
     public getUsersAsync(HomeActivity home) {
         this.home = home;
     }
+    public getUsersAsync(HomeFragment home) { this.homeF = home; }
+
 
     //doInBackground is the method called when a JsonAccessor.execute("url") is called
     @Override
     protected ArrayList<User> doInBackground(Integer...params)
     {
         try{
+            if ( params[0] == 1)
+            {
+                selected =1;
+            }
+            else if ( params[0] == 2)
+            {
+                selected =2;
+            }
+            else
+            {
+                selected = 999;
+            }
```

Comment:
Its dumb, but its setting a variable depending on what fragment getUsersAsync() is called from.
It does this so it knows where to invoke getUsersResponse()